cosmwasm/cli-utils.js

```diff
@@ -41,6 +41,10 @@ const addAmplifierOptions = (program, options) => {
         addExecuteProposalOptions(program);
     }
 
+    if (options.registerItsChainOptions) {
```

Comment:
we should avoid making addAmplifierOptions be a catch all for every cmd type. Only common amplifier options should be defined here. single use ones should be defined within the script itself, or have another add* variant if reuse is needed I think.

Suggested something in the script below